ruby-challenges.rb

```diff
@@ -11,6 +11,24 @@
 # Expected output: '221 is odd'
 
 # Pseudo code:
+# input: an integer
+# out put: a string that tells whether the integer is even or odd
+# create a method that takes in an integer
+# write an conditional statement using unless 
+# unless the condition .even? is false, the output is always "#{integer} is odd"
+# use else to return the other scenerio when the condition is true and the integer is even
+
+def even_or_odd (integer)
+    unless integer.even? 
+        "#{integer} is odd"
+    else 
+        "#{integer} is even"
+    end
+end
+p even_or_odd (reposts1)
+p even_or_odd (reposts2)
+p even_or_odd (reposts3)
```

Comment:
You can do it with the `unless`, but more conventional to use `if`.  Either way it works!  Again, thinking outside of the box. :)